BWV 1004 Violin Partita II D minor/5-ciaccona.ly

```diff
@@ -13,6 +13,7 @@
     
     \time 3/4
     \key d \minor
+    \set Score.currentBarNumber = #2
```

Comment:
Sure about this? I particularly do not have a **strong** opinion on this but I would rather let lilypond take care of it (I bet different editions will use different numbering... or is there a standard?). Either way, if you want to have this in, please, also adjust the numbering of the measures in the file because they won't match.

Reply:
To my knowledge, every edition, that includes bar numbering, numbers the first bar as bar number 1.

Breitkopf und Härtel, 1879 : no numbering.
Bärenreiter, 1958 : number 1.
Bärenreiter, 2014 : number 1.
Serge Blanc, 2017: number 1.

This being said, there is an alternative to using "\set Score.currentBarNumber = #2": to start the first measure with "s4" instead of "\partial 2". With this, lilypond calculates the number in accordance with general agreement.

In any case, I agree that the comments must be updated. So the change is so extended that we may want a dedicated pull request.